test/joiner_team_index_by_field_test.rb

```diff
@@ -33,7 +33,6 @@ def add_team_member(member_hash, private: false)
     def impl
       joiner_impl = JoinerImpl.new @site
       joiner_impl.data.merge! joiner_impl.collection_data
-      joiner_impl.promote_or_remove_data
```

Comment:
why is this being removed? looks like it's still being called in `lib/team_api/joiner.rb`